smsclient_core/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+SMS client core
+===========
+
+This module provide base function to send sms. It not works in stand alone, it needs a least one provider module
```

Comment:
s/It not works in stand alone, it needs/It is not meant to work alone and needs
add period